UpsertedProfileOrchestrator/__tests__/handler.test.ts

```diff
@@ -251,4 +254,854 @@ describe("UpsertedProfileOrchestrator", () => {
       }
     );
   });
+
+  it("should call UpdateSubscriptionFeedActivity when profile is created", async () => {
+    const expectedQueueName = "queue_name" as NonEmptyString;
+    const upsertedProfileOrchestratorInput = UpsertedProfileOrchestratorInput.decode(
+      {
+        newProfile: {
+          ...aRetrievedProfile,
+          email: aEmailChanged, // Email changed to start the EmailValidationProcessOrchestrator
+          isInboxEnabled: true // Enable inbox to start the SendWelcomeMessagesActivity
+        },
+        oldProfile: undefined,
```

Comment:
This scenario should doesn't exists. When the profile is created `isInboxEnabled` is `false`. The next update with the  setting of `acceptedTosVersion` change it to `true`

Reply:
Ok I'll remove it